sdk/digitaltwins/pom.xml

```diff
@@ -23,7 +23,7 @@
         <dependency>
           <groupId>com.azure</groupId>
           <artifactId>azure-digitaltwins-core</artifactId>
-          <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-digitaltwins-core;current} -->
+          <version>1.1.5</version> <!-- {x-version-update;com.azure:azure-digitaltwins-core;current} -->
```

Comment:
Since the artifact does not yet exist, would this not fail? Given the XML comment next to it, is this not a place where the value is used from version_client.txt?

Reply:
I believe this was auto-updated by the PrepareRelease tooling. At any rate the build succeeds in CI.